experimental/samples/runtime/dotnet/echo-bot/echo-bot/runtime.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
We should be able to update this to reflect the latest changes to single adapter, etc.

Reply:
Yes.  This is dependent on an older version of the Runtime package.  The templates and samples will have to change when the newer Runtime package is published.